src/app/beer_garden/api/http/handlers/v1/garden.py

```diff
@@ -109,17 +111,38 @@ class GardenAPI(BaseHandler):
           - Garden
         """
 
-        response = await self.client(
-            Operation(
-                operation_type="GARDEN_UPDATE",
-                args=[
-                    garden_name,
-                    SchemaParser.parse_patch(
-                        self.request.decoded_body, from_string=True
-                    ),
-                ],
-            )
-        )
+        patch = SchemaParser.parse_patch(self.request.decoded_body, from_string=True)
+
+        for op in patch:
+            operation = op.operation.lower()
+
+            if operation in ["initializing", "running", "stopped", "block"]:
+                response = await self.client(
+                    Operation(
+                        operation_type="GARDEN_UPDATE_STATUS",
+                        args=[garden_name, operation.upper()],
+                    )
+                )
+            elif operation == "heartbeat":
+                response = await self.client(
+                    Operation(
+                        operation_type="GARDEN_UPDATE_STATUS",
+                        args=[garden_name, "RUNNING"],
+                    )
+                )
+            elif operation == "config":
+                response = await self.client(
+                    Operation(
+                        operation_type="GARDEN_UPDATE_CONFIG",
+                        args=[
+                            garden_name,
+                            SchemaParser.parse_garden(op.payload, from_string=True),
```

Comment:
Have you tested this? I don't believe a patch operation has a payload field.

Reply:
You are right, it should have been `op.value`